packages/xml-views-tooltip/src/tooltip.ts

```diff
@@ -75,12 +72,15 @@ function findUI5NodeByElement(
     return undefined;
   }
   const nameByKind = isOpenName
-    ? astNode.syntax.openName?.image
-    : astNode.syntax.closeName?.image;
+    ? /* istanbul ignore next */
```

Comment:
Add a comment that the openName/closeName cannot be undefined here because otherwise the ast position visitor wouldn't return their types